packages/vm/src/state/baseStateManager.ts

```diff
@@ -75,6 +76,10 @@ export abstract class BaseStateManager {
 
     this._checkpointCount = 0
 
+    if (opts.customPrecompiles) {
+      this._customPrecompiles = opts.customPrecompiles
+    }
```

Comment:
These custom precompiles properties and options would need to be removed as well.